app/support/products/scheduling_support.rb

```diff
@@ -25,6 +25,11 @@ def admin_reservations
     self.reservations.admin
   end
 
+  def started_reservations
+    self.purchased_reservations
+      .merge(Reservation.relay_in_progress)
```

Comment:
Should this be only getting the reservations with actual_start_at set (relays) or should using reserve_start_at?